cmd/vicadmin/vicadm.go

```diff
@@ -311,8 +318,11 @@ func httpEntry(name string, res *http.Response) (entry, error) {
 
 func (path urlReader) open() (entry, error) {
 	defer trace.End(trace.Begin(string(path)))
-
-	res, err := http.Get(string(path))
+	timeout := time.Duration(2 * time.Second)
+	client := http.Client{
+		Timeout: timeout,
```

Comment:
It's only a few K or megs of log files over a network internal to the vSphere environment.  The alternative is for users to wait potentially a very long time if every collector hangs.